demo/packages/tabs/stories/10-announcement.stories.mdx

```diff
@@ -0,0 +1,229 @@
+import { Story, Meta, html, Preview } from '@open-wc/demoing-storybook';
+
+<Meta title="Intro/Announcement" />
+
+# ING Bank open sources lion
```

Comment:
A title like `ING Bank open sources Lion - A library for building performant, accessible & flexible Web Components` might be more catching to the eye when you scroll past it on HackerNews/Medium